### ISSUE TITLE
`welspell_config()` hangs RStudio if closed via [x] button

### ISSUE THREAD

**Issue by GegznaV** [nevrome/wellspell.addin#17]
If *wellspell* add-in configuration window is closed via `[x]` button as shown in the figure, it hangs RStudio and the only option is to kill the program via Task Manager.

![image](https://user-images.githubusercontent.com/12725868/66125283-37f10e80-e5ef-11e9-8ee3-dad961aee3dd.png)

![image](https://user-images.githubusercontent.com/12725868/66125532-d7160600-e5ef-11e9-9ec7-9252683eeaf0.png)

**Comment by nevrome** [comment]
I know this problem, but I have no idea how to solve it. I tried some things, but was not successful. Maybe we have to ask [here](https://github.com/rstudio/miniUI/issues).

**Comment by GegznaV** [comment]
Yes, an issue on *miniUI* GitHub page should be opened. Would you do this, @nevrome?

**Comment by GegznaV** [comment]
Or maybe a minimal reproducible example should be created to make sure that the issue is related to *miniUI*. If it is not a bug in *miniUI*, Stack overflow can be an option too.

**Comment by GegznaV** [comment]
I played a bit with the add-in. The bug appears only if the configuration window opens if grammar or spelling checking were called. And the bug is not present if configuration add-in is called directly or if function `set_config()` is called directly. So, I came to the conclusion that the problem is present in RStudio. What I found in the  *addins.dcf* file is that grammar and spelling checking add-ins are called as non-interactive ones. However, they open the interactive shiny app to choose the language of the dictionaries. It seems that setting `Interactive: true` solves the issue. 

See my changes in pull request #20.
